framework/scripts/cluster_control.py

```diff
@@ -175,6 +175,19 @@ def calculate_seconds(start_time, end_time):
                     f"{node_info['status']['last_sync_agentgroups']['n_synced_chunks']}.\n"
             msg2 += f"                Permission to synchronize agent-groups: " \
                     f"{node_info['status']['sync_agent_groups_free']}.\n"
+        else:
+            if node_info['status']:
+                msg2 += "        Status:\n"
+                msg2 += "            Agents reconnect:\n"
+                msg2 += f"                Current phase: {node_info['status']['agents_reconnect']['current_phase']}.\n"
+                msg2 += f"                Last workers stability check: " \
```

Comment:
Just as a suggestion:

```suggestion
                msg2 += f"                Last stability check: " \
```